data/data/bootstrap/systemd/units/release-image-pivot.service.template

```diff
@@ -3,11 +3,7 @@
 Description=Pivot bootstrap to the OpenShift Release Image
 Wants=release-image.service
 After=release-image.service
-{{- if .BootstrapInPlace }}
```

Comment:
I dropped this condition only as a temporary measure to test the remaining parts and because I could not find a proper go-template variable for agent-based. But we should consider the consequences before removing the condition completely. Do we always have a kubelet.service? Even in cases where we do not use agent-based and bootstrap-in-place?

Reply:
`release-image-pivot.service` is only used on OKD, so OCP will not be impacted at all. 

[`Before=` will not let systemd fail if a service does not exist](https://serverfault.com/questions/1062205/does-systemd-fail-if-a-dependency-in-after-requires-doesnt-exist), so in case where `bootkube.service` or `kubelet.service` does no harm.

When `bootkube.service` or `kubelet.service` do exist, we would always want to ensure that `release-image-pivot.service` is started first because it might reboot the system or change `/usr` in the live iso use case.

That is why I think it is safe to drop the `.BootstrapInPlace` check and unconditionally ask systemd to launch `release-image-pivot.service` before `bootkube.service` and `kubelet.service`.

Reply:
There could be other scenarios where it may be useful to differentiate between abi and not-abi cases, for this reason I've prepared https://github.com/openshift/installer/pull/7508 that could be helpful to support the current scenario, and it may be handy for future ones